src/i18n/locales/en/translation.json

```diff
@@ -634,6 +634,19 @@
       },
       "success": "Withdraw request of {{amount}} sent successfully to {{sender}}"
     },
+    "lnurlredeem": {
+      "title": "Redeem bitcoin",
+      "input": {
+        "label": "LNURL-withdraw code",
+        "placeholder": "LNURL..."
+      },
+      "actions": {
+        "withdraw": "Withdraw"
+      },
+      "qrcode": {
```

Comment:
QR code translations should not be duplicated. Could you look if these can be moved to a common place?

Reply:
resolved